tests/r/FormatRBearTest.py

```diff
@@ -15,10 +14,11 @@
 if(TRUE){
 x=1  # inline comments
 }else{
-x=2;print('Oh no... ask the right bracket to go away!')}
+x=2;print("Oh no... ask the right bracket to go away!")}
 """.splitlines(keepends=True)
 
 
 FormatRBearTest = verify_local_bear(FormatRBear,
                                     valid_files=(good_file,),
-                                    invalid_files=(bad_file,))
+                                    invalid_files=(bad_file,),
+                                    tempfile_kwargs={"suffix": ".R"})
```

Comment:
hey we talked on gitter and you said this is not needed, so you might as well remove it as settled :)